packages/runtime/container-runtime/src/gcSweepReadyUsageDetection.ts

```diff
@@ -17,28 +18,35 @@ import { oneDayMs } from "./garbageCollection";
  * Feature Gate Key -
  * How many days between closing the container from this error (avoids locking user out of their file altogether)
  */
-export const skipClosureForXDaysKey = "Fluid.GarbageCollection.Dogfood.SweepReadyUsageDetection.SkipClosureForXDays";
+export const skipClosureForXDaysKey = "Fluid.GarbageCollection.Dogfood.SweepReadyUsageDetection.SkipForXDays";
 
 /**
  * LocalStorage key (NOT via feature gate / monitoring context)
  * A map from docId to info about the last time we closed due to this error
  */
 export const closuresMapLocalStorageKey = "Fluid.GarbageCollection.Dogfood.SweepReadyUsageDetection.Closures";
 
+interface SweepReadyUsageDetectionConfig {
+    interactiveClient?: "close" | "crashOnLoadOnly";
+}
+
 /**
  * Feature gate key to enable closing the container if SweepReady objects are used.
  * Value should contain keywords "interactiveClient" and/or "summarizer" to enable detection in each container type
  */
-const sweepReadyUsageDetectionSetting = {
-    read(config: IConfigProvider) {
+ const sweepReadyUsageDetectionSetting = {
+    read(config: IConfigProvider): SweepReadyUsageDetectionConfig {
         const sweepReadyUsageDetectionKey = "Fluid.GarbageCollection.Dogfood.SweepReadyUsageDetection";
         const value = config.getString(sweepReadyUsageDetectionKey);
         if (value === undefined) {
-            return { interactiveClient: false, summarizer: false };
+            return { };
         }
         return {
-            interactiveClient: value.includes("interactiveClient"),
-            summarizer: value.includes("summarizer"),
+            interactiveClient: value.includes("interactiveClientClose")
```

Comment:
Why are we doing an includes check rather than a comparison check?

Reply:
So you can indicate multiple modes (e.g. "interactiveClientClose, summarizer") or something.

Reply:
I'd prefer 1 flag per feature, it doesn't really matter to me, but it may confuse other developers as this is a new pattern.

Reply:
For the record:  We discussed it as a FC and came to this decision together :P